src/common/lib/api.ts

```diff
@@ -17,11 +17,11 @@ import {
 } from "./cache";
 import utils from "./utils";
 
-interface AccountInfoRes {
+export interface AccountInfoRes {
+  balance: { balance: string | number };
```

Comment:
I think we have to make a decision here. Always string or always number. (used to be string)

Reply:
Currently we get `0` when balance it's zero but a `string` when there's a balance. Will check if we can adjust the `0` balance handling.

Reply:
+1 we need to check amount handling. I am not sure if number is good enough? what's with big.js or something like that?

Reply:
Hm, maybe pre TS nobody noticed:

- LND  
      `0` as strings:    
      ![image](https://user-images.githubusercontent.com/1016218/172527424-e8ce543c-06e0-45a9-a3f8-82f78aa8c2a9.png)

- LNDHUB
      `0` as number:
       ![image](https://user-images.githubusercontent.com/1016218/172527479-4c421efc-9e66-484a-9dbd-13bf991327f0.png)

Should we keep `number | string` for now or should we check the connectors and parse anything `number` to be ` string`?

Reply:
On the other hand this is how `AccountInfo` is typed currently:  
![image](https://user-images.githubusercontent.com/1016218/172544025-bb3ac316-0b5b-4b8e-a552-7d5571701e25.png)

Reply:
> +1 we need to check amount handling. I am not sure if number is good enough? what's with big.js or something like that?

Not too experienced with this. As long as we don't have an issue here I think we're good for now. In this case `balance` is always sats, right? So we won't end up with an issue because of i.e. `0.1324422` sats?